app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -32,7 +32,7 @@ import { ToneValue } from './ToneValue';
 import { isWorkOrderStatus } from './utils';
 
 //Difference ratio
-const threshold = 0.1;
+//const threshold = 0.1;
```

Comment:
Looks good - Quick question, do we still need the commented out code? Here and below?

Reply:
Yes, I will be working on these later on to check if the summary is different from transcript.

Reply:
You'll need to remove them in the following PR.